### ISSUE TITLE
Broken existing test_data_set and model (bertsquad-8.onnx, arcfaceresnet100-8.onnx)

### ISSUE THREAD

**Issue by jcwchen** [onnx/models#431]
# Bug Report

### Which model does this pertain to?
66 existing test_data_set are broken and 2 models (bertsquad-8.onnx, arcfaceresnet100-8.onnx
) seems broken as well

### Describe the bug
Describe the problem with the model and the results you are seeing.
While running ORT backend test (https://github.com/onnx/models/pull/384) with all models from ONNX model Zoo, 66 models cannot pass by original test_data_set and 2 models fail even with creating test_data_set

For broken test_data_set, there are couple of reasons:

1. (1 fail) bertsquad-10.onnx: output mismatch
2. (5 fails) yolov4.onnx, faster_rcnn_R_50_FPN_1x.onnx, ssd_mobilenet_v1_10.onnx, MaskRCNN-10.onnx, roberta-base-11.onnx
: empty output name
3. (Other 60 fails) Same as https://github.com/onnx/models/issues/403. Many input lack name or the original name is incorrect.

Two models fail because:
1. bertsquad-8.onnx: Gather node: indices element out of data bounds, idx=2 must be within the inclusive range [-2,1]
2. vision/body_analysis/arcface/model/arcfaceresnet100-8.onnx: BatchNormalization node: Invalid input scale: NumDimensions() != 3

### Reproduction instructions

**System Information**  

OS Platform and Distribution (*e.g. Linux Ubuntu 16.04*): Mac
ONNX version (*e.g. 1.6*):  1.8
Backend/Runtime version (*e.g. ONNX Runtime 1.1, PyTorch 1.2*):  latest ORT (ort-nightly)
